sdk/core/ts-http-runtime/scripts/azure-diff.ts

```diff
@@ -0,0 +1,182 @@
+import { parseArgs } from "node:util";
```

Comment:
Honestly I really appreciate that you chose to use `node` APIs instead of pulling in a dependency like `yargs` or something (which is, what I probably would have done) 👍